user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+.. _user-guide-intro-log-in-edit-profile:
+
+Log In, View Your User Page, and Set Up Your Profile
+====================================================
+
+To get to your OroCRM account, enter the OroCRM instance url.
+
+.. image:: ./img/intro/login.png
+
+.. hint::
+
+    The *"login using google"* option may not be available for your organization, check with your OroCRM Administrator.
+    
+    
+.. _user-guide-getting-started-log-in:
+
+Log-In
+------
+
+Your administrator will determine this but typically upon your user account creation you will receive an email with your 
+login credentials. In order to log in to the system for the first time, type in the username and password specified in 
+the email. In order to save your credentials on a device, check the *"Remember me on this computer"* box before you log 
+in.
+
+.. caution::
+
+    It is strictly recommended to change your password immediately after the first login. You can learn how to do it in 
+    the :ref:Changing Your Password section `<user-guide-getting-started-profile-password>` below.
+
+If you have forgotten the credentials, click the *"Forgot your password?"* link and follow the wizard to restore them.
+
+
+Log In Using Your Google Credentials
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the Google Single Sign-On capability has been enabled for your organization, you will see the *"login using google"* 
```

Comment:
Edit: If the Google Single Sign-On capability has been enabled for your organization, you will see the *"login using Google"*